go_modules/spec/dependabot/go_modules/file_updater_spec.rb

```diff
@@ -91,6 +91,44 @@
       expect(updated_files.find { |f| f.name == "go.sum" }).to_not be_nil
     end
 
+    context "with a repo not found" do
```

Comment:
From the fixtures it seems like it's not the repo of the dependency we're specifying that can't be found, but rather a transitive dependency that is private, is that right?

Should we communicate that in the test more clearly?

Reply:
How about "with an indirect dependency from an unreachable repo"?